docs/manual/comparison_with_rdatatable.rst

```diff
@@ -0,0 +1,330 @@
+``DT[y>2 & v>5]``                   ``df[(f.y>2) & (f.v>5), :]``                Compound logical expressions
+``DT[!2:4]`` or ``DT[-(2:4)]``      ``df[[0, slice(4, None)], :]``              All rows other than rows 2,3,4
+``DT[order(x), ]``                  | ``df.sort("x")`` or                       Sort by column ``x``, ascending
+                                    | ``df[:, :, sort("x")]``
+``DT[order(-x)]``                   | ``df.sort(-f.x)`` or                      Sort by column ``x``, descending
+                                    | ``df[:, :, sort(-f.x)]``
+``DT[order(x, -y)]``                | ``df.sort(x, -f.y)`` or                   Sort by column ``x`` ascending, ``y`` descending
+                                    | ``df[:, :, sort(x, -f.y)]``
+===============================   ============================================ ==================================================
+
+Note the use of the ``f`` symbol when performing computations or sorting in descending order. You can read more about :ref:`f-expressions`.
+
+Selecting Columns
+------------------
+
+=========================================== ============================================ ==============================================
+RDatatable                                           Datatable                                                  Action
+=========================================== ============================================ ==============================================
+``DT[, .(v)]``                               ``df[:, 'v']``                              Select column ``v``
+``DT[, .(x,v)]``                             ``df[:, ['x', 'v']]``                       Select multiple columns
+``DT[, .(m = x)]``                           ``df[:. {"m" : f.x}]``                      Rename and select column
+``DT[, .(sv=sum(v))]``                       ``df[:, {"sv": dt.sum(f.v)}]``              Sum column ``v`` and rename as ``sv``
+``DT[, .(v, v*2)]``                          ``df[:, [f.v, f.v*2]]``                     Return two columns, ``v`` and ``v`` doubled
+``DT[, 2]``                                  ``df[:, 1]``                                Select the second column
+``DT[, ncol(DT), with=FALSE]``               ``df[:, -1]``                               Select last column
+``DT[, .SD, .SDcols=x:y]``                   ``df[:, f["x":"y"]]``                       Select columns ``x`` through ``y``
+``DT[ , .SD, .SDcols = !x:y]``               | ``df[:, [name not in ("x","y")``          Exclude columns ``x`` and ``y``
+                                             |          ``for name in df.names]]`` or
+                                             | ``df[:, f[:].remove(f['x':'y'])]``
+``DT[ , .SD, .SDcols = patterns('^[xv]')]``  | ``df[:, [name.startswith(("x","v"))``     Select columns that start with x or v
+                                             |          ``for name in df.names]]``
```

Comment:
When I render this table it seems that there is a horizontal scroll bar that appears. We should try to avoid it, but I'm not sure if the page width is a hard constraint on readthedocs. @st-pasha any ideas?

<img width="760" alt="Screen Shot 2020-09-11 at 11 18 18 AM" src="https://user-images.githubusercontent.com/35204136/92959345-7fe3fc80-f420-11ea-80cd-b10094611aac.png">

Reply:
@oleksiyskononenko not sure how I would go about affecting the width. maybe @st-pasha has some ideas?

Reply:
Yeah, that's something that is out of your control here and should be addressed globally. I guess you have a reasonable amount of content presented.

Reply:
@samukweku While we're trying to solve the problem with rendering of this table, I thought it could look more logical if we  move the "Action" column to the first place, i.e. the column order of "Action/data.table/datatable". What do you think?

Reply:
I put the `Action` column at the end, because it follows a read from left to right approach; the user familiar with R sees the data.table code, sees the python code, and then sees the action ... in this scenario, the `Action` column is more like comments without the `#`

Reply:
I could probably find a better word than `Action`, ... maybe `Explanation` ?

Reply:
Yes, may be "Operation" or may be even just remove the caption. It is pretty obvious just reading the text, that it is a description of the particular operation on data.

Regarding the column order, I agree that for an experienced R user it is ok to start with the R's code right away, but for all the others it will require some time to understand what's going on. If we do it as "Action/data.table/datatable", then, reading it from left to right, a person will immediately get a feeling of what kind of functionality we're going to present and then see the examples.